titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/admission/JobRuntimePredictionSelectors.java

```diff
@@ -58,18 +59,30 @@ public static JobRuntimePredictionSelector best(Map<String, String> selectionMet
     /**
      * Select prediction with the lowest confidence level which is above the provided confidenceThreshold.
```

Comment:
javadoc needs to be updated

Reply:
a few words about how `sigma` is our estimation of a stddev for the distribution of predicted runtimes